packages/stories-utils/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@chanzuckerberg/stories-utils",
```

Comment:
Does `stories-utils` make sense as a package name?

Eventually this package will probably have
- prepareStory
- snapshotTestStories
- getStories

Reply:
Another option could be storybook-utils 🤷‍♀️

Reply:
For sure. Although I'm trying to keep the word "storybook" out of the naming, because these are intended to be used outside of Storybook (eg - in Jest tests).

Reply:
That makes sense 😄